packages/mui-lab/src/Masonry/Masonry.js

```diff
@@ -47,6 +47,7 @@ export const getStyle = ({ ownerState, theme }) => {
     boxSizing: 'border-box',
     '& > *': {
       boxSizing: 'border-box',
+      flexShrink: '0 !important',
```

Comment:
I would not use `!important`, unless it's absolutely needed. If we consider this offers better DX, we should document as it would be not possible to override unless using `!important` - customization is always one of the things that is hardest if using bigger specificity.